client/src/components/Loader/Loader.tsx

```diff
@@ -1,9 +1,16 @@
 import React, { FC } from 'react';
 import styles from './styles.module.scss';
 
-export const Loader: FC = () => {
+export enum LoaderStyles {
+  search = 'searchContainer',
+  main = 'loaderContainer'
+}
+interface LoaderProps {
+  className: string;
+}
+export const Loader: FC<LoaderProps> = ({ className }) => {
   return (
-    <div className={styles.loaderContainer}>
+    <div className={styles[className]}>
```

Comment:
i'd recommend to move searchContainer style to the component that uses Loader and needs it to be customized. and pass this className using props. in this case your loader should look like this:
```suggestion
    <div className={`${styles.loaderContainer} ${className}`}>
```